kitsune/sumo/static/sumo/js/libs/jquery.ajaxupload.js

```diff
@@ -9,137 +9,138 @@
  * if you want to avoid that).
  */
 
-import dialogSet from "sumo/js/upload-dialog";
```

Comment:
What are the changes in this file?

Reply:
Pretty sure I deleted and replaced the file content after filling it full of debug/log code that I commited then corrected. So...just space I believe.

Reply:
The only pertinent change is from jquery's json parser to the now standard issue JSON.parse